lib/loader.js

```diff
@@ -53,6 +53,9 @@ class VASTLoader extends EventEmitter {
         const vast = parse(body, {
           noSingleAdPods: this._options.noSingleAdPods
         })
+        if (vast == null) {
+          throw new VASTLoaderError(101, null, uri)
+        }
```

Comment:
Is it feasible to distinguish between
 - VAST 100 Error = Invalid XML (to test with empty file, JSON, file, plain text).
 - VAST 101 Error = Current test with `<InvalidAd>` child.